src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_image_builder_commands.py

```diff
@@ -235,7 +235,7 @@ def test_image_builder_template_validator(self, resource_group):
             'loc': 'westus',
             'tmpl_01': 'template01',
             'tmpl_02': 'template02',
-            'img_src': LINUX_IMAGE_SOURCE,
+            'img_src': 'CANONICAL:UBUNTUSERVER:18_04-LTS-GEN2:latest',
```

Comment:
Why don't we consider changing `LINUX_IMAGE_SOURCE` directly to `CANONICAL:UBUNTUSERVER:18_04-LTS-GEN2:latest`?

Reply:
Some feature require gen2 image.